.github/workflows/stale-issues.yml

```diff
@@ -13,12 +12,15 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           stale-issue-message: "This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days. If a Velero team member has requested log or more information, please provide the output of the shared commands."
           close-issue-message: "This issue was closed because it has been stalled for 5 days with no activity."
-          days-before-issue-stale: 30
-          days-before-issue-close: 5
+          days-before-issue-stale: 30 # 60
```

Comment:
we keep the days of the other job?

Reply:
My opinion is to use the longer setting.